src/main/java/org/terasology/scenario/internal/systems/ConvertEntitySystem.java

```diff
@@ -53,24 +53,34 @@
 
 import java.util.Map;
 
+/**
+ * System that recursively takes a logic entity and develops a list of serialised strings that allow for recreation of the entity
+ * following the same order of the list(Makes sure that the list always goes down in in depth, not a skip of branches that would result in
+ * wanting to satisfy an entity that doesn't yet exist
+ */
 @RegisterSystem(RegisterMode.CLIENT)
 public class ConvertEntitySystem extends BaseComponentSystem {
 
     /**
      * Most entities will follow a similar method, but to leave the system open for future inclusions with new event/action/conditions
-     * it is designed with each specific component(the unique component of a logic prefab)
+     * it is designed with each specific component(the unique component of a logic prefab). All are based off the component that indicates what
+     * type an entity is.
+     *
+     * Follows a pattern of [PREFAB]prefabName{key name for entity argument}[VALUE]value of the component
      */
 
+    private final String prefab = "[PREFAB]";
+    private final String value = "[VALUE]";

```

Comment:
Should be static as it is the same for everything and since it is then a constant it should be upper case. Maybe you also want to call it PREFAB_TYPE_ID or something like that to indicate that it is not a prefab but an id/marker